bin/configurable-http-proxy

```diff
@@ -44,6 +44,9 @@ args
     .option('--no-include-prefix', "Don't include the routing prefix in proxied requests")
     .option('--insecure', "Disable SSL cert verification")
     .option('--host-routing', "Use host routing (host as first level of path)")
+    .option('--statsd-host <host>', 'Host to send statsd statistics to')
+    .option('--statsd-port <port>', 'Port to send statsd statistics to', parseInt)
+    .option('--statsd-prefix <prefix>', 'Prefix to use for statsd statistics')
```

Comment:
We could, but that's somewhat nonstandard (from my experience, at least!). The prefix is also not a URL prefix, but a prefix to use for the metrics - so if prefix is 'tools.paws.chp' (as it is in my case), the metrics would be tools.paws.chp.request.web, etc. So putting them in URL notation doesn't make sense, IMO.